release/__init__.py

```diff
@@ -302,8 +302,9 @@ def make_stable_artifacts(cache_repository_url):
     with logger.scope("Building packages"):
         try:
             all_completes = do_build_packages(cache_repository_url)
-        except pkgpanda.build.BuildError as ex:
-            logger.error("Building packages: {}".format(ex))
+        except pkgpanda.build.BuildError:
+            logger.exception("An error was encountered while building packages")
```

Comment:
It's still nice to have the error message formatted into the log stream if possible here. Sometimes can be hard to find the exception otherwise.